packages/components/src/components/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -142,10 +142,16 @@ export const CheckboxGroup: FC<CheckboxGroupProps> = ({
           const isObject = typeof item === 'object'
           const value = isObject ? item.value || item.text : item
 
+          const accessibilityValue = useA11yListPosition(
+            index + 1,
+            items.length,
+          )
+
           return (
             <Fragment key={`checkbox-group-item-${index}`}>
               <Checkbox
                 label={item}
+                accessibilityValue={accessibilityValue}
```

Comment:
Could be updated to `aria-valuetext` to align with newer/more web-like RN props.